gradle/scala.gradle

```diff
@@ -49,12 +49,18 @@ test {
 sourceSets {
     main {
         scala {
-            srcDir "src/main/scala_$scalaBaseVersion"
+            srcDirs += ["src/main/scala_$scalaBaseVersion", "src/main/java"]
+        }
+        java {
+            srcDirs = []
         }
     }
```

Comment:
Should we put java sources under java source dir ?
```main {
          scala {
                   srcDir "src/main/scala_$scalaBaseVersion"
         }
         java {
             srcDir "src/main/java"
          }
      }
``` 

And just to point out - `srcDir "src"` and `srcDirs += []` is equal, so we can stick wit any of those